ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/table.utils.ts

```diff
@@ -2,6 +2,8 @@ import { gql } from '@apollo/client';
 import { min, parse } from 'date-fns';
 import sortBy from 'lodash/sortBy';
 import uniq from 'lodash/uniq';
+import pluralize from 'pluralize';
```

Comment:
Not a request for change, as this is the current standard in the code base, but what is your opinion on using standalone `'pluralize'` versus the `pluralize()` helper function that ships with PatternFly?

Reply:
Super question! I will add item to discuss in team meeting.